### PR TITLE
feat: `AppWindow` positioning and sizing

### LINKED ISSUE #17098: Add support for `AppWindow.Move` method
### What would you like to be added

Add support for `AppWindow.Move` method

### Why is this needed

To move the window

### For which platform

_No response_

### Anything else we need to know?

Reference https://github.com/unoplatform/uno/issues/8978

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): closes #17098

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.Runtime.Skia.X11/X11XamlRootHost.cs [hunk 1]
```diff
@@ -84,16 +84,20 @@ public X11XamlRootHost(X11WindowWrapper wrapper, Window winUIWindow, XamlRoot xa
...
-			_ = XLib.XResizeWindow(TopX11Window.Display, TopX11Window.Window, (int)size.Width, (int)size.Height);
-			resizeCallback(size);
-		};
 		_closingCallback = closingCallback;
 		_focusCallback = focusCallback;
 		_visibilityCallback = visibilityCallback;
+		_configureCallback = configureCallback;
+
+		_configureTimer = new DispatcherTimer();
+		_configureTimer.Interval = new TimeSpan(1000 / 16);
```
Comment (ajpinedam) [inline src/Uno.UI.Runtime.Skia.X11/X11XamlRootHost.cs:93]: what does this constant represent (1000 / 16)? 

Hz?

File: src/Uno.UI.Runtime.Skia.Gtk/UI/Controls/GtkWindowWrapper.cs [hunk 2]
```diff
@@ -134,6 +135,7 @@ private void OnHostSizeChanged(object? sender, Windows.Foundation.Size size)
 	{
 		Bounds = new Rect(default, size);
 		VisibleBounds = Bounds;
+		Size = new Windows.Graphics.SizeInt32() { Height = (int)size.Height, Width = (int)size.Width };
```
Comment (ajpinedam) [inline src/Uno.UI.Runtime.Skia.Gtk/UI/Controls/GtkWindowWrapper.cs:138]: Could it be used here `size.ToSizeInt32()` ?

**Review by ajpinedam — APPROVED** [review]
Looks good!

File: src/Uno.UI.Runtime.Skia.X11/X11XamlRootHost.cs (hunk 1 above)
Comment (MartinZikmund) [reply src/Uno.UI.Runtime.Skia.X11/X11XamlRootHost.cs:93]: Not sure cc @ramezgerges

Comment (ramezgerges) [reply src/Uno.UI.Runtime.Skia.X11/X11XamlRootHost.cs:93]: This checks and updates window sizing /positioning details 60 times a second.

**Pull request merged**